tensorflow/lite/micro/kernels/detection_postprocess.cc

```diff
@@ -350,6 +350,57 @@ void DecreasingPartialArgSort(const float* values, int num_values,
       [&values](const int i, const int j) { return values[i] > values[j]; });
 }
 
+template <typename Compare>
+void insertion_sort(int* start, int* end, Compare compare) {
+  for (int* i = start; i != end; ++i) {
+    std::rotate(std::upper_bound(start, i, *i, compare), i, i + 1);
+  }
+}
+
+template <typename Compare>
```

Comment:
do these functions need to be templated? If it is not required right now, then I would prefer to not have it.

Reply:
It is needed to pass the compare function object around, which is used in a few places and it in that way the lambda expression can be defined in one place, like it was before. So in that sense I prefer to have them... ok?

Reply:
sounds ok to me.